universal.c

```diff
@@ -216,12 +216,7 @@ Perl_sv_isa_sv(pTHX_ SV *sv, SV *namesv)
     if(!SvROK(sv) || !SvOBJECT(SvRV(sv)))
         return FALSE;
 
-    /* This abuse of gv_fetchmeth_pv() with level = 1 skips the UNIVERSAL
-     * lookup
-     * TODO: Consider if we want a NOUNIVERSAL flag for requesting this in a
```

Comment:
Unless the `-1` part here really is relating to that NOUNIVERSAL behaviour, please leave this TODO comment in. It's a reminder to me that we still want that behaviour.

Reply:
I thought the entire comment was one idea - I'll put back the TODO part of it.

Reply:
Done - in commit ec8dc484c1